locales/en/translation.json

```
@@ -74,8 +74,10 @@
   },
   "main": {
     "failed-to-start-header": "Kando failed to start",
+    "show-settings": "Show settings",
```

I think the other entries are capitalized.

```suggestion
    "show-settings": "Show Settings",
```

---

Thank you for your suggestion! I updated the code.